src/integrations/mod.rs

```diff
@@ -1,10 +1,273 @@
+use std::path::PathBuf;
+use std::sync::Arc;
+use indexmap::IndexMap;
+use serde_json::json;
+use tracing::{info, warn};
+use tokio::sync::{Mutex as AMutex, RwLock as ARwLock};
+
+use crate::global_context::GlobalContext;
+use crate::integrations::integr::Integration;
+use crate::integrations::integr_chrome::IntegrationChrome;
+use crate::integrations::integr_github::IntegrationGitHub;
+use crate::integrations::integr_gitlab::IntegrationGitLab;
+use crate::integrations::integr_pdb::IntegrationPdb;
+use crate::integrations::integr_postgres::IntegrationPostgres;
+use crate::tools::tools_description::Tool;
+use crate::yaml_configs::create_configs::{integrations_enabled_cfg, read_yaml_into_value};
+
+pub mod sessions;
+pub mod process_io_utils;
 pub mod integr_github;
 pub mod integr_gitlab;
 pub mod integr_pdb;
 pub mod integr_chrome;
-pub mod sessions;
-pub mod process_io_utils;
 pub mod integr_postgres;
+mod integr;
+
+
+// hint: when adding integration, update: 
+// DEFAULT_INTEGRATION_VALUES, INTEGRATION_ICONS, integrations_paths, validate_integration_value, load_integration_tools, load_integration_schema_and_json
+
+
+pub const DEFAULT_INTEGRATION_VALUES: &[(&str, &str)] = &[
+    ("github.yaml", integr_github::DEFAULT_GITHUB_INTEGRATION_YAML),
+    ("gitlab.yaml", integr_gitlab::DEFAULT_GITLAB_INTEGRATION_YAML),
+    ("pdb.yaml", integr_pdb::DEFAULT_PDB_INTEGRATION_YAML),
+    ("postgres.yaml", integr_postgres::DEFAULT_POSTGRES_INTEGRATION_YAML),
+    ("chrome.yaml", integr_chrome::DEFAULT_CHROME_INTEGRATION_YAML),
+];
```

Comment:
None of that is necessary. You already have "trait Integration", there is no reason at all to have any other dicts for each integration.